### PR TITLE
Choose monsters

### PR DESCRIPTION
**Pull request by mml2222**
Issue #9 

title: which monsters might help you on your journey? --> Connected with create tag
list of monsters
ability to select >1 monster
writes in database to UserMonsters --> usersID--> PredictedMonsters --> ProjectID

### CODE REVIEW

File: tags/create.tag [hunk 1]
```diff
@@ -40,20 +44,32 @@
...
+          this.showPickMonsters = true;
+
+
         }
         else{
           throw new Error('User is not signed in - should not see create tag');
         }
       }
     }
+    //add lisiner to pass project id to pickMonster tag
```
Comment (lrbloch) [inline tags/create.tag]: @mml2222 did you mean to put in a listener here or is this an old comment that should be deleted?

File: tags/pickMonster.tag [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        <div class="form-group">
+          <div class="col-md-4" each={ monsterItem, i in myMonsters}>
+            <label class="btn btn-info">
+              <img src={ monsterItem.img } alt={ monsterItem.name } class="img-thumbnail img-check { check: monsterItem.pick }" onclick={ parent.toggle }>
+              <input type="checkbox" name={ monsterItem.name } id={ monsterItem.id } class="hidden">
+            </label>
+          </div>
+        </div>
+      </div>
+      <button type="button" class="btn btn-success" onclick={ selectMonster }>Next</button> <!-- need to add link to next page -->
```
Comment (lrbloch) [inline tags/pickMonster.tag:20]: @mml2222 a helpful thing to put in the comment for something that needs to be done is "todo" - most IDEs will find all comments with "todo" in them and show them to you as to-do items!
(you don't need to change this, I just wanted to let you know for the future)

File: tags/pickMonster.tag [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  let database = firebase.firestore()
+  var monsterRef = database.collection('Monsters');
+  this.myMonsters = [];
+  let refPredictedMonsters;
+
+  //read monster assets from database
+  monsterRef.onSnapshot(function (snapshot) {
+    var monsters = [];
+    snapshot.forEach(function (doc) {
+      this.monster = true;
```
Comment (lrbloch) [inline tags/pickMonster.tag]: @mml2222 what is `this.monster` used for?

File: tags/pickMonster.tag [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      monsters.push(doc.data());
+    })
+    that.myMonsters = monsters;
+    that.update();
+  });
+
+  // receives projectId
+  observer.on('project:created', (curProject) => {
+    this.projectId = curProject;
```
Comment (lrbloch) [inline tags/pickMonster.tag:45]: OH I didn't understand how observers worked until now! this is great :)

**Review by lrbloch — COMMENTED** [review]
looks great! I just listed a few questions/notes, but well done! I learned a lot from reading your code.
(This PR is for Issue #9)

File: tags/create.tag (hunk 1 above)
Comment (mml2222) [reply tags/create.tag]: Yes, it an old comment! I will delete it

File: tags/pickMonster.tag (hunk 2 above)
Comment (mml2222) [reply tags/pickMonster.tag:20]: Ok! Thank you!

File: tags/pickMonster.tag (hunk 3 above)
Comment (mml2222) [reply tags/pickMonster.tag]: Your right thank you. I will delete the line.

File: tags/pickMonster.tag (hunk 4 above)
Comment (mml2222) [reply tags/pickMonster.tag:45]: Yay! <3

**Review by lrbloch — APPROVED** [review]

**Pull request merged**